test/test_packages.py

```diff
@@ -62,7 +62,6 @@
 
 # List of packages that cannot be tested in a standard way
 EXCLUDED_PACKAGES = [
-    "tini",
```

Comment:
This is system package, it won't be tested at all, so we don't need to exclude it.